larky/pom.xml

```diff
@@ -129,6 +130,12 @@
             <version>${org.bouncycastle.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.jpos</groupId>
```

Comment:
Do we have to worry about jpos' license?

Reply:
not sure but it seems to as we are safe
http://www.jpos.org/license